snake_config: ["end_effector", "link_module", "joint_module", "link_module", "end_effector"]
d: [0, 0, 0, 0, 0, 0, 0]
a: [0.1, 0.6, 0.15, 0.1, 0.1, 1.3, 0]
alpha: [0, 0, 0.705, 1.5708, -1.5708, 0, 0]
theta: [0, 0, 0, 0, 0, 0, 0]
base_frame: 2

visualization: true
joint_topic: "joint_angles"
odom_topic: "odometry"
transform_topic: "kinematics"
